os/tools/mkbinheader.py

```diff
@@ -84,18 +84,38 @@
     else : # Not supported.
         bin_type = 0
 
+    # Calculate RAM size
+    # Dynamic RAM size : user input of argv[6]
+    # Static RAM size : Extract from size command in linux(ONLY for elf type)
+    if bin_type == BIN :
+        os.system('size ' + tinyara_path + ' > static_ram_file')
+    elif bin_type == ELF :
+        os.system('size ' + file_path + ' > static_ram_file')
+    else : #Not supported.
+
+    ram_fp = open("static_ram_file", 'rb')
+    ram_fp.readline()
+    ram_array = ram_fp.readline()
+    ram_fp.close()
+    os.remove('static_ram_file')
+    size_array = ram_array.split('\t')
+    static_ram_size = size_array[3]
```

Comment:
What is `3`?

Reply:
In size command on linux, 4th value is the summation of text, data and bss.
We will use this value for elf type.
I leave the comment in the code also.

Reply:
Let's use readable definition instead of 3.